src/libraries/System.Utf8String.Experimental/ref/System.Utf8String.Experimental.csproj

```diff
@@ -1,15 +1,36 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- disable warnings about obsolete APIs -->
     <NoWarn>$(NoWarn);0809;0618</NoWarn>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.0;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
@joperezr do we want to cross compile this for full framework as well so that whenever used there, the netstandard shims are not part of the closure?

Reply:
Good point, yes that would be ideal given that this does have a netstandard2.0 asset so ideally we will also want to add a netfx asset here. That said, this is not something that has to be done now, I still have to do a pass to do this on all release/3.1 packages, and then will do a pass on runtime/master where packages like this would get picked up.

Reply:
The `netstandard2.0` version of this package references `System.Memory` v`4.5.3` which doesn't ship a netfx asset. So I don't believe this would actually help.

Reply:
As part of this all-up path that we are doing and I mentioned above, we actually fixed System.Memory package already and will soon ship version 4.5.4 which will have a netfx asset. That is precisely why we are going from the bottom up in terms of order, so that is why this one  will eventually get fixed at which point all of its dependencies will already have a netfx asset.

Reply:
Sounds good. We can fix this package when we do the rest of the packages in master then.